tests/test_com_aux.py

```diff
@@ -8,6 +8,7 @@
 ##########################################################################
 
 import logging
+import queue
```

Comment:
Update your branch pytest have evolved

Reply:
Yep, just forgot to rebase after fetch upstream